applications/asset_tracker_v2/doc/modules.rst

```diff
@@ -0,0 +1,26 @@
+.. _asset_tracker_v2_modules:
+
+Application modules
+###################
+
+The Asset Tracker v2 application uses a set of modules.
+See :ref:`asset_tracker_v2_internal_modules` for more information.
```

Comment:
The tag for the architecture page should probably be renamed to `asset_tracker_v2_architecture`, especially since we now have another one called `asset_tracker_v2_modules`. I don't think it is linked elsewhere, and the current name is not precise.

The linking sentence could maybe be clarified a bit, like:
```suggestion
See :ref:`asset_tracker_v2_architecture` for more information on how the application is structured.
```